doc/code-generator/build.gradle

```diff
@@ -0,0 +1,138 @@
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+
+plugins {
+  id 'java'
+  id 'org.openapi.generator'
+  id 'maven-publish'
+}
+
+group = "com.wci.termhub"
+version = "1.0.0-SNAPSHOT"
+
+// Input Specification locations. When using the inputFile, make sure you've updated
+// the doc with the changes you want to generate
+def inputURL = 'https://api.terminologyhub.com/terminology/doc/v3/api-docs'
+def inputFile = '$projectDir/src/th-swagger.yaml'
+
+repositories {
+  mavenCentral()
+}
+
+sourceCompatibility = 17
+targetCompatibility = 17
+
+test {
+  useJUnitPlatform()
+}
+
+/**
+ * Builds a Java client by default: https://openapi-generator.tech/docs/generators/java/
+ * To use a local file, replace {remoteInputSpec.set(inputURL)} with input {inputSpec.set(inputFile)}
+*/
+//tasks.register('buildJavaSdk', GenerateTask) {
+//  generatorName.set("java")
+//  remoteInputSpec.set(inputURL)
+//  outputDir.set(layout.buildDirectory.dir("java").get().getAsFile().getAbsolutePath())
+//  apiPackage.set("com.wci.termhub.api")
+//  invokerPackage.set("com.wci.termhub.api.invoker")
+//  modelPackage.set("com.wci.termhub.api.model")
+//  configOptions.set([
+//      "dateLibrary": "java8",
+//      "library": "restclient",
+//      "serializationLibrary": "jackson",
+//      "hideGenerationTimestamp": "false"
+//  ])
+//}
+
+/**
+ * Build a python client by default: https://openapi-generator.tech/docs/generators/python/
+ * To use a local file, replace {remoteInputSpec.set(inputURL)} with input {inputSpec.set(inputFile)}
+ */
+tasks.register('buildPythonSdk', GenerateTask) {
+  generatorName.set("python")
+  remoteInputSpec.set(inputURL)
+  outputDir.set(layout.buildDirectory.dir("python").get().getAsFile().getAbsolutePath())
+  apiPackage.set("com.wci.termhub.api")
+  invokerPackage.set("com.wci.termhub.api.invoker")
+  modelPackage.set("com.wci.termhub.api.models")
+  configOptions.set([
+      "packageName": "python-examples",
+      "hideGenerationTimestamp": "false",
+  ])
+}
+
```

Comment:
Remove commented out code

Reply:
The commented out code is a placholder for the next languages we are going to implement. Since we know they will be supported and I was able to investigate all of them at once I wrote the code to just comment back in and generate the code immediately.